packages/js/product-editor/src/components/block-editor/block-editor.tsx

```diff
@@ -271,8 +313,34 @@ export function BlockEditor( {
 		);
 	}
 
+	const formTemplateSelectValues = productForms?.map( ( form ) => ( {
+		label: form.title.raw,
+		value: String( form.id ),
+	} ) );
+
 	return (
 		<div className="woocommerce-product-block-editor">
+			{ isProductEditorTemplateSystemEnabled && (
+				<div style={ { margin: '32px', width: '250px' } }>
```

Comment:
Considering it is temporary and behind a feature flag, it is OK to keep it like this for now.

Reply:
yes, it should be properly addressed when working on this [one](https://github.com/woocommerce/woocommerce/issues/48236)